README.md

```diff
@@ -154,8 +183,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
-
+There are 80 different valid sentences.
```

Comment:
# -2

Actually, it's infinite, because there can be an indefinite number of adjectives.